### PR TITLE
SDS-3067: Document Flexibility & Serenity PIM update/migration process

### PR DESCRIPTION
**Pull request by ClementGautier**
# Description

Long time ago we were asked to put a bit of doc about PIM upgrad process on Flexibility and Serenity offer.
That's not a lot but at least it cover the need
cf. https://akeneo.atlassian.net/browse/SDS-3067

# Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

### CODE REVIEW

File: cloud_edition/serenity_mode/docs/pim_updates_and_migrations.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PIM Updates and Migrations
+==========================
+
+As the Serenity offer is designed with simplicity in mind, you don't have to do anything to
+benefit from the latest patch and releases. The Akeneo team will take care of the upgrades
```
Comment (LudovicTOURMAN) [inline cloud_edition/serenity_mode/docs/pim_updates_and_migrations.rst]: `patches and releases` or `patch and release`

File: cloud_edition/serenity_mode/docs/pim_updates_and_migrations.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+PIM Updates and Migrations
+==========================
+
+As the Serenity offer is designed with simplicity in mind, you don't have to do anything to
+benefit from the latest patches and releases. The Akeneo team will take care of the upgrades
+as quickly as possible.
+
+You will get the latest stable (minus alpha, beta or pre-released) version available.
```
Comment (mr-prud) [inline cloud_edition/serenity_mode/docs/pim_updates_and_migrations.rst]: 'minus' for  me minus sound like the symbol, perhaps 'not'  could be without ambiguity

Comment (LudovicTOURMAN) [reply cloud_edition/serenity_mode/docs/pim_updates_and_migrations.rst]: Maybe, just removing it as it sounds normal that alpha, beta, pre-released are not stable releases.

**Comment by ClementGautier** [comment]
It got proof-read by a beloved English colleague.

**Pull request merged**